sphinx_needs/builder.py

```diff
@@ -92,3 +92,58 @@ def build_needs_json(app: Sphinx, _exception: Exception) -> None:
     needs_builder = NeedsBuilder(app)
     needs_builder.set_environment(env)
     needs_builder.finish()
+
+
+class NeedumlsBuilder(Builder):
+    name = "needumls"
+
+    def write_doc(self, docname: str, doctree: nodes.document) -> None:
+        pass
+
+    def finish(self) -> None:
+        env = unwrap(self.env)
+        needumls = env.needs_all_needumls.values()
+
+        for needuml in needumls:
+            if needuml["save"]:
```

Comment:
Maybe some out put to the user would be nice.
E.g. A line like "Storing needuml to {file}"